### DISCUSSION TITLE
Introducing conditional formatting to Tables in Budibase! 🎨

### DISCUSSION THREAD

**Discussion by jamie-budibase** [Budibase/budibase#14236] [Announcements]
You can now create rules to dynamically change a cell's text or background color based on its value. For example, you could set a rule to turn a cell’s background green when the value is set to "Done.” This improves the “at a glance” experience of your table, making it easy to scan for important information.

Rules are set at the column level, and based on these rules, you can customize formatting at the row and cell levels.

Upgrade to the latest version of Budibase today to try it for yourself today!

**Comment by rvt** [comment]
Hey Jamie,

I looked at it.. and this is very usefull..
However, I really really miss formatting of cells, for example:

Date's: There is no way to format them to the correct context of what the table is showing
Numbers: I think it's use full to add a prefix/suffix perfect would be using a table cell formatter 
Strings: Sometimes we just need to shorten them 'eg "Users wants to ....." 
I can possible see other types that require formatting.

A workaround is ofcourse adding a Static/Dynamic formula field in the table but that get's unpractical very quickly...

The old table has that, I am yet trying to figure out to hack that in my newly created Budibase app :)

**Reply by jamie-budibase** [reply]
Hey RVT,

I'm glad you're finding it useful, and thanks for the feedback. This is great for us when prioritizing future updates.  

I've passed your message on to our engineering team, so hopefully we'll be able to implement some of your suggestions. Keep an eye out for updates!

Thanks,
Jamie
